pkg/daemon/metrics.go

```diff
@@ -159,8 +159,8 @@ func addFlagsForMonitor(nodeProfile *ptpv1.PtpProfile) {
 		}
 
 		if !strings.Contains(*nodeProfile.Ptp4lOpts, "--summary_interval") {
-			glog.Info("adding --summary_interval 1 to print summary messages to stdout for ptp4l to use prometheus exporter")
-			*nodeProfile.Ptp4lOpts = fmt.Sprintf("%s --summary_interval 1", *nodeProfile.Ptp4lOpts)
+			glog.Info("adding --summary_interval 0 to print summary messages to stdout for ptp4l to use prometheus exporter")
```

Comment:
Changing summary_interval to 0 will result a different format of ptp4l output message. will it affect how message is parsed?

Reply:
you are right I will revert this and work on another PR to support both the summary and the regular output.

In Telco they will like to use the normal output without the summary.